scripts/publish.js

```diff
@@ -15,7 +15,13 @@ function getTaggedVersion() {
  * @param {string} tag
  */
 function publish(dir, tag) {
-  execSync(`pnpm publish ${dir} --access public --tag ${tag}`, {
+  let args = ["--access public", `--tag ${tag}`];
+  if (tag === "experimental") {
+    args.push(`--no-git-checks`);
+  } else {
+    args.push("--publish-branch release-next");
+  }
```

Comment:
By default, `pnpm` will only publish from `main`.  Turn off this check when publishing an experimental from `release-experimental-*` and change the allowed publish branch to `release-next` for normal releases.